modules/ossm-control-plane-cli.adoc

```diff
@@ -1,6 +1,7 @@
-// This module is included in the following assemblies:
-//
-// * service_mesh/v2x/ossm-create-smcp.adoc
+////
+This module is included in the following assemblies:
+* service_mesh/v2x/ossm-create-smcp.adoc
+////
```

Comment:
**PEER REVIEWER**: This format seems to be more common in the Service Mesh docs.

Reply:
Luv ya man, meenit. :-D